client/app/scripts/superdesk-legal-archive/views/legal_archive.html

```diff
@@ -66,7 +66,8 @@
                     <div class="filter-trigger" ng-click="openAdvanceSearch = true" ng-class="{hidden: openAdvanceSearch}"><i class="icon-filter-large"></i></div>
                     <div sd-legal-item-sortbar></div>
                     <div sd-pagination data-items="items"></div>
-
+                    <div class="searchbar-flex-handler"></div>
```

Comment:
Seams that this is not related to related items, so at least a different commit is better.